save.go

```diff
@@ -156,8 +157,10 @@ func nixName(goImportPath string) string {
 	return strings.Replace(parts[len(parts)-1], ".", "-", -1)
 }
 
-func findDeps(name, gopath string, testImports bool) ([]string, error) {
+func findDeps(name, gopath string, testImports bool, buildTags []string) ([]string, error) {
 	ctx := build.Default
+	ctx.BuildTags = buildTags
```

Comment:
I'm not sure about this code:

It's mutating a global object if I understand well, so I don't really like this idea and believe it would be better to clone `build.Default` before or something, but I'm lacking experience in Go do to this properly.